src/WebexSDKAdapter.js

```diff
@@ -0,0 +1,16 @@
+import RoomsSDKAdapter from './RoomsSDKAdapter';
+
+export default class WebexSDKAdapter {
+  /**
+   * Creates a new instance of the WebexSDKAdapter.
+   * The data source for this adapter comes from an authenticated instance of the SDK.
+   *
+   * @param {Object} sdk The primary sdk the adapter will be using.
+   * @todo implement and collect sub-adapters here in the constructor
+   */
+  constructor(sdk) {
+    this.sdk = sdk;
```

Comment:
don't think we would need this anymore. Had to add it to prevent displaying some eslint errors.

Reply:
I'm using `this.sdk` in the rooms adapter so we definitely need it.